avionics/common/src/radio.cpp

```diff
@@ -186,7 +189,7 @@ void Radio::sendSingleSensor(const uint32_t time, uint8_t id, float data) {
     Radio::addSubpacket(std::move(buf.packet));
 }
 
-void Radio::sendState(const uint32_t time, uint8_t state_id) {
+void Radio::sendState(const uint32_t time, uint16_t state_id) {
     PacketBuffWriter buf;
 
     buf.write(0x1D); // id
```

Comment:
The actual implementation needs to be updated - this is left over from when state was sent as a single sensor

Reply:
Should be fixed now, but hard to test given that @DSep is doing the software side. Let me know if anything else needs to be fixed.

Reply:
Main thing is it ought to be rebased on top of dev before continuing